test/alternative-text.doc.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE HTML>
+<body>
+<h1><img src="https://placekitten.com/200/300" alt="Little cat"></h1>
+
+<p><img src="https://placekitten.com/300/200" alt="Secret cat"></p>
+
+<!-- more about cats -->
+<h2>Text &amp; entities</h2>
+
+<p>More &quot;entities&quot;</p>
+
+<h2><input type="IMAGE" SRC="https://placekitten.com/500/500" ALT="Click me"></h2>
```

Comment:
:) but does this mean a network request is actually made during the test run? This is probably undesirable. Can you use a local image? Might need to include it in the `resources` here: https://github.com/h5o/h5o-js/blob/master/buster.js#L13